src/lib/components/icons/apple-icon-dark.svelte

```diff
@@ -8,11 +8,8 @@
   height={size}
   width={size}
   class={classes}
-  viewBox="0 0 24 24"
+  viewBox="0 0 24 20"
```

Comment:
I had to play around with the properties for the social icon SVGs to get them to align consistently. Again I took them from Wikimedia, but if we know of a better place to get them I'm all ears!

Reply:
All I can get from Facebook is a set of official PNGs, but no SVGs. I think Google is similar but need to check along with Apple. Seeing as we are referencing other brands now in this project, would we prefer official PNGs, or publicly sourced (wikimedia), SVGs as defaults? I think official PNGs would probably be best, but didn't want to work against the patterns in the rest of the app.

Reply:
I'd say the Bootstrap icons are likely the safest, most consistent source of icons: https://icons.getbootstrap.com. Other notable are https://www.flaticon.com and, if you super desperate, https://worldvectorlogo.com.

Reply:
I went with Bootstrap SVGs in the end.